python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -896,6 +896,114 @@ def corr(self, method="pearson", min_periods=1):
 
         return res
 
+    def cov(self, min_periods=None, ddof=1):
```

Comment:
I don't see code to handle `min_periods=None`, and it doesn't look like that case is tested.

Reply:
I see you asked a question about this in the PR description. pandas uses `min_periods=0` by default. This is a little bit hidden -- here is the snippet defining that: https://github.com/pandas-dev/pandas/blob/99f61783c55ae9a791b83610312b65581954bb70/pandas/_libs/algos.pyx#L342-L345

The `nancorr` function here is called here if any of the values are `NaN`: https://github.com/pandas-dev/pandas/blob/a769e389343dec0d59df3022a5f0b503ffd42dfc/pandas/core/frame.py#L9625-L9633

You can set the default value to `0` in the signature.